dirs/dirs.go

```diff
@@ -176,9 +176,9 @@ func SetRootDir(rootdir string) {
 	SnapDataDir = filepath.Join(rootdir, "/var/snap")
 	SnapDataHomeGlob = filepath.Join(rootdir, "/home/*/snap/")
 	SnapAppArmorDir = filepath.Join(rootdir, snappyDir, "apparmor", "profiles")
+	SnapConfineAppArmorDir = filepath.Join(rootdir, snappyDir, "apparmor", "snap-confine.d")
```

Comment:
Can we please fix the name for this directory to "snap-confine" while we have time (no .d)? That convention doesn't make much sense here.

Reply:
Done